testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -1274,6 +1274,8 @@ public void verifyFT85() throws IOException, InterruptedException {
     /**
      * To verify the that Save Inbox call Details API request should not succeed with content provided for only 1
      * subscription pack and not even a place holder for second Pack details
+     *
+     * //TODO : FT case to be modified .Refer NMS -182
```

Comment:
FT description to be updated in next version of FT doc.
I have conveyed this to Kanchan.

On Thu, Jun 25, 2015 at 9:21 PM, Frank Huster <notifications@github.com>
wrote:

> In
> testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java
> <https://github.com/motech-implementations/mim/pull/425#discussion_r33270653>
> :
>
> > @@ -1274,6 +1274,8 @@ public void verifyFT85() throws IOException, InterruptedException {
> >      /**
> >       * To verify the that Save Inbox call Details API request should not succeed with content provided for only 1
> >       * subscription pack and not even a place holder for second Pack details
> > +     *
> > +     * //TODO : FT case to be modified .Refer NMS -182
>
> OK, so I assume you will change the comment for this method (which
> currently doe snot match the code) when the FT description is updated?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/425/files#r33270653>.
>